projects/media-viewer/src/lib/viewers/image-viewer/image-viewer.component.ts

```diff
@@ -147,4 +152,8 @@ export class ImageViewerComponent implements OnInit, OnDestroy, OnChanges {
   getImageHeight(img) {
     return this.rotation % 180 !== 0 ? img.offsetWidth + 15 : img.offsetHeight + 15;
   }
+
+  toggleCommentsSummary() {
+    this.toolbarEvents.toggleCommentsSummary(!this.toolbarEvents.showCommentSummary.getValue());
```

Comment:
why are we doing this here and not in the header component?

Reply:
because we want the header to be a stateless component and its parent component will be holding the state...